### DISCUSSION TITLE
Pkl demo?

### DISCUSSION THREAD

**Discussion by tomByrer** [live-codes/livecodes#507] [Ideas]
Apple just released a new meta configuration language:
https://pkl-lang.org/

Runs on Groovy.  Seems to be [gaining attention](https://youtu.be/lAxXWYAIt4k).

I thought of suggesting to them to use Live-codes for their docs, but I'm unsure how hard it would be to add support here?

**Comment by hatemhosny** [comment]
Thanks @tomByrer 
I'm not sure we can run Pkl in the browser (on the client-side), which is a requirement for language support in LiveCodes. 
In addition, it seems to be a DSL that is targeting configuration. I'm not sure this fits the criteria for having a language that can produce a meaningful output to a web page. E.g. we do not have JSON as a separate language. 

The criteria for adding languages to LiveCodes are summarized here: https://github.com/live-codes/livecodes/blob/develop/docs/docs/contribution/adding-languages.md

Let me know what you think.

**Reply by tomByrer** [reply]
Ah Ok, I wasn't sure of your scope.
Feel free to lower-prioritize/close as you wish!
